### ISSUE TITLE
[New] Clarify Accepted "Studio" Links for Scenes

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#27]
We need to better clarify what is and isn't acceptable for the "studio link" field for scenes. In particular, there are concerns about using JAVLibrary links in this field instead of original studio websites. We could also use this section to explain guidance around public links vs. members-only links, semi-official affiliate links, network vs. sub-studio domains, and "canonical" link structure (including a `www.`, MindGeek slugs, etc.)

(Related to this topic is similar guidance around performer links — what constitutes an "official website" in particular — and a request for better guidance on how to source pictures and performer/scene info for JAV.)

This is a situation where we have already formed a shared consensus around best practices, so we should be able to add "unofficial" sections covering this subject without waiting for "official" approval first.

**Comment by DoctorD1501** [comment]
I agree that 3rd party databases (like JAVLibrary) should not be used as the studio link. It might be OK to have these kinds of links in the comments section of an edit when scraping and adding a scene for the first time or if we could make a new kind of link for them, that would be OK too.

**Comment by AdultSun** [comment]
JAVLibrary is a piracy concern, so we won't be able to add it as a separate link type on StashDB.
